coredns/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM alpine as runtime
```

Comment:
Is there a reason this is in the coredns/ subdir instead of the top level directory?

Reply:
top-level directory is related to the package fanout. If we put all files from this directory to top-level directory then we face with `Multiple packages in directory` error.

Reply:
Also, I think we should not add `main.go` file for `fanout` module because it can break the mechanism of Caddyserver for loading external plugins. Information about`fanout` plugin https://github.com/coredns/coredns.io/pull/190 has been merged to Coredns website and soon this plugin can be used by people.